src/components/general/Stepper/styles.less

```diff
@@ -8,6 +8,7 @@
     .stepPane {
         display:flex;
         flex-direction: row;
+        flex-wrap: wrap;
```

Comment:
Couldn't this end up taking a lot of space? Has it been discussed with Lotte?

Reply:
I guess so if you have a lot of steps.

Reply:
I don't know if you need to revert. This will make the stepper usable on mobile, I just don't know if it's a good long term solution.